tasks/e2e-installs.sh

```diff
@@ -184,6 +173,10 @@ CI=true yarn test
 # Eject...
 echo yes | npm run eject
 
+# Temporary workaround for https://github.com/facebook/create-react-app/issues/6099
+rm yarn.lock
```

Comment:
This workaround is due to https://github.com/facebook/create-react-app/issues/6679 (which looks like a duplicate of https://github.com/facebook/create-react-app/issues/6099). Without this, build/test almost always fails due to a missing @babel plugin. Like is discussed in the issues, I haven't found a way to repro locally (at least not consistently).